skietbaan_fe/src/actions/types.js

```diff
@@ -43,7 +45,8 @@ export const UPDATE_SELECTED_COMPETITION = "UPDATE_SELECTED_COMPETITION";
 export const UPDATE_SELECTED_GROUP = "UPDATE_SELECTED_GROUP";
 
 //groups types and constants
-export const URL = "https://api.skietbaan.retrotest.co.za";
+//https://api.skietbaan.retrotest.co.za"
+export const URL = "http://localhost:50859";
```

Comment:
should this still point to localhost when it's going into dev?

Reply:
resolved